app/controllers/taxpayer/UpdateTaxpayerController.scala

```diff
@@ -94,8 +95,9 @@ class UpdateTaxpayerController @Inject()(
 
           for {
             updatedAnswers <- Future.fromTry(userAnswers.set(UpdateTaxpayerPage, value))
-            _              <- sessionRepository.set(updatedAnswers)
-          } yield Redirect(navigator.nextPage(UpdateTaxpayerPage, mode, updatedAnswers))
+            cleanAnswers   <- Future.fromTry(updatedAnswers.remove(WhatIsTaxpayersStartDateForImplementingArrangementPage)) // TODO test when userAnswers are properly supplied
```

Comment:
Is this still pending?

Reply:
Yes, this has moved to ticket DAC6-513. Once we connect the taxpayers journey to the task list (DAC6-503), we can inject the `DataRetrievalAction` and have the set of pages we need to test this.